packages/cli-platform-ios/src/tools/getArchitecture.ts

```diff
@@ -0,0 +1,14 @@
+import {readFile} from 'fs-extra';
+import path from 'path';
+
+export default async function getArchitecture(iosSourceDir: string) {
+  try {
+    const project = await readFile(
+      path.join(iosSourceDir, '/Pods/Pods.xcodeproj/project.pbxproj'),
+    );
+
+    return project.includes('-DRCT_NEW_ARCH_ENABLED=1');
```

Comment:
@cipolleschi is this fine if we use that to detect new arch?

Reply:
yes, this should be enough.